src/java/htsjdk/samtools/SAMFileWriterFactory.java

```diff
@@ -293,16 +291,39 @@ public SAMFileWriter makeSAMOrBAMWriter(final SAMFileHeader header, final boolea
         return makeBAMWriter(header, presorted, outputFile);
     }
 
+    /**
+     *
+     * Create a SAM, BAM or CRAM writer based on examination of the outputFile extension.
+     * For CRAM files, read names and tags are preserved.
```

Comment:
That's exactly what I would like to be able say (that by default its completely lossless). But I don't know for sure if its true. Hence #380. For now I'll remove these since I think you're right that they're not helpful without a more comprehensive statement.